### PR TITLE
update IGNORE_EXISTED_INDEX

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

**Review by whitewum — DISMISSED** [review]

File: docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md [hunk 1]
```diff
@@ -19,6 +19,28 @@ prop_value_list:
...
+prop_name_list:
+   [prop_name [, prop_name] ...]
+prop_value_list:
+   [prop_value [, prop_value] ...]   
+```
+- `IGNORE_EXISTED_INDEX`：Indexes will not be updated after insert vertex. We recommend that using `IGNORE_EXISTED_INDEX` on the first import, which can greatly improve performance.
+
+  !!! danger
+
+        Existing indexes will not be deleted when using the `IGNORE_EXISTED_INDEX`, so incorrect data may be read during lookups. You need to rebuild the index, for more information, see [REBUILD INDEX](../3/../14.native-index-statements/4.rebuild-native-index.md。
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md:39]: why do we need to rebuild the existing index?

Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md:39]:
```suggestion
        Existing indexes will not be deleted when using the `IGNORE_EXISTED_INDEX`, so incorrect data may be read during lookups. You need to rebuild the index. For more information, see [REBUILD INDEX](../3/../14.native-index-statements/4.rebuild-native-index.md。
```

**Review by abby-cyber — APPROVED** [review]

File: docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md [hunk 2]
```diff
@@ -52,6 +74,37 @@ nebula> CREATE TAG IF NOT EXISTS t1();
...
+ nebula> INSERT VERTEX person(id) VALUES "100":(1), "200":(1);
+ # Lookup vertices which id is "1", both 100 and 200 will satisfy.
+ nebula> LOOKUP ON person WHERE person.id == 1 YIELD id(vertex) as id;
+ +-------------+
+ | id          |
+ +-------------+
+ | "100"       |
+ | "200"       |
+ +-------------+
+ # Insert vertices with `IGNORE_EXISTED_INDEX` which VID "200", id is "2".
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md:89]:
```suggestion
 # Insert a vertex with `IGNORE_EXISTED_INDEX`. The VID of the vertex is "200" and the value of the property id is with "2".
```

File: docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md [hunk 3]
```diff
@@ -52,6 +74,37 @@ nebula> CREATE TAG IF NOT EXISTS t1();
...
+ nebula> INSERT VERTEX IGNORE_EXISTED_INDEX person(id) VALUES "200":(2);
+ # Lookup vertices which id is "1", both 100 and 200 will satisfy.
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md:91]:
```suggestion
 # Query vertices, the value of the property id is "1", both 100 and 200 will be returned.
```

File: docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md [hunk 4]
```diff
@@ -52,6 +74,37 @@ nebula> CREATE TAG IF NOT EXISTS t1();
...
+ nebula> LOOKUP ON person WHERE person.id == 1 YIELD id(vertex) as id;
+ +-------------+
+ | id          |
+ +-------------+
+ | "100"       |
+ | "200"       |
+ +-------------+
+ # Lookup vertices which id is "2", only 200 will satisfy.
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md:99]:
```suggestion
 # Query vertices, the value of the property id is "2", only 200 will be returned.
```

File: docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md (hunk 1 above)
Comment (foesa-yang) [reply docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md:39]: Lookup error data may occur if indexes are not rebuilt

Comment (whitewum) [reply docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md]: > 

by design

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md [hunk 5]
```diff
@@ -19,6 +19,28 @@ prop_value_list:
...
+INSERT VERTEX [IF NOT EXISTS] [IGNORE_EXISTED_INDEX] [tag_props, [tag_props] ...]
+VALUES VID: ([prop_value_list])
+tag_props:
+  tag_name ([prop_name_list])
+prop_name_list:
+   [prop_name [, prop_name] ...]
+prop_value_list:
+   [prop_value [, prop_value] ...]   
+```
+- `IGNORE_EXISTED_INDEX`：Indexes will not be updated after insert vertex. We recommend that using `IGNORE_EXISTED_INDEX` on the first import, which can greatly improve performance.
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md:35]:
```suggestion
- `IGNORE_EXISTED_INDEX`：Indexes will not be updated after a vertex is inserted. We recommend using `IGNORE_EXISTED_INDEX` on the first import, which can greatly improve performance.
```

**Review by abby-cyber — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**